components/pedigree/resources/src/main/resources/pedigree/pedigreeEditorParameters.js

```diff
@@ -118,7 +118,10 @@ define([], function(){
         layoutRelativeOtherWidth: 2,
         layoutScale: { xscale: 12.0, yscale: 8 },
         maxPrintPreviewPaneHeight: 600,
-        minPrintPreviewPaneHeight: 250
+        minPrintPreviewPaneHeight: 250,
+        howerboxHighlightColor: "green",
```

Comment:
yes. But since it passed two code reviews I merged as is. Will try not to forget to fix the typo later